akka-http/src/main/resources/reference.conf

```diff
@@ -39,5 +39,5 @@ akka.http.routing {
 
   # Fully qualified config path which holds the dispatcher configuration
   # to be used by FlowMaterialiser when creating Actors for IO operations.
-  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher}
+  file-io-dispatcher = ${akka.io.tcp.file-io-dispatcher} // TODO point to akka.stream.file-io-dispatcher instead?
```

Comment:
I'm not sure if trying to adjust the size based on number of cores are relevant for such a pool or if it is more appropriate to define a fixed size?